cmd/keeper/cmd/keeper.go

```diff
@@ -288,7 +292,13 @@ func (p *PostgresKeeper) createPGParameters(db *cluster.DB) common.Parameters {
 		parameters[k] = v
 	}
 
-	parameters["listen_addresses"] = fmt.Sprintf("127.0.0.1,%s", p.pgListenAddress)
+	var lst = make([]string, 0)
+
+	if p.pgListenAddress != "*" {
+		lst = []string{"127.0.0.1"}
+	}
```

Comment:
just noticed that listening on localhost is probably not needed anymore by the stolon keeper since we use unix sockets. Perhaps it's useful for people doing local connections (local backups scripts etc...) so we could keep it to avoid possible regression.

Reply:
yes i think so, but I wanted to keep as much functionality as possible.
I your are agree, I will remove it :)